pkg/sdn/plugin/node.go

```diff
@@ -79,13 +83,16 @@ func NewNodePlugin(pluginName string, osClient *osclient.Client, kClient *kclien
 }
 
 func (node *OsdnNode) Start() error {
-	var err error
-	node.networkInfo, err = getNetworkInfo(node.registry.oClient)
+	cn, err := node.osClient.ClusterNetwork().Get(osapi.ClusterNetworkDefault)
 	if err != nil {
 		return fmt.Errorf("Failed to get network information: %v", err)
 	}
+	node.networkInfo, err = parseNetworkInfo(cn.Network, cn.ServiceNetwork)
+	if err != nil {
+		return err
+	}
```

Comment:
This all could be a helper function that returns cn, netinfo, and err.  It's also used in proxy.go.